aries-site/src/pages/templates/filtering.mdx

```diff
@@ -0,0 +1,81 @@
+import { Example } from '../../layouts';
+import {
+  FilteringCards,
+  FilteringLists,
+  FilteringWithRangeInput,
+  FilteringWithSelect,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/filtering/FilteringCards.js"
+  showResponsiveControls={['laptop', 'mobile']}
+  template
+>
+  <FilteringLists />
+</Example>
+
+## What is filtering?
+
+Filtering allows data to be narrowed down around a set of specified parameters. When users may not have a specific record in mind but are seeking to group data around a certain set of attributes, filtering should be used to allow the user to narrow the data and take action on relevant records.
+
```

Comment:
We should add in a lower fidelity example or .png which shows where filter controls live in relation to the content + how the controls live:
[
[ Filter controls: [Primary Filter] [Primary Filter] [More Filters (filter icon)] ]
[ Record summary / count ]
[ Content being filtered ]
]

Could just add this as a comment placeholder for now and create an issue.